### DISCUSSION TITLE
ipv6?

### DISCUSSION THREAD

**Discussion by hemna** [n7tae/mrefd#34] [General]
Hello,
  I'm just learning about M17 and looking into the feasability of running mrefd myself.  In the readme it says there has to be some open inbound ports available.  Unforuntately for me, my ISP only has ipv6 addresses that are public routable, and my ipv4 address is NAT'd at the ISP, so I can't open those ports for ipv4.   Does mrefd support ipv6?

thanks

**Comment by n7tae** [comment]
Yes. In fact there is already an  IPv6-only mrefd running in Australia, [M17-IP6](http://m17-ip6.vkradio.com:3000/#/).

You can run mrefd at home for educational purposes, but mrefd should really be run on a commercial server.

**Comment by n7tae** [comment]
And a lot of hosting companies will offer discounts if you don't want an IPv4 address.
